pepper-apis/pom.xml

```diff
@@ -74,4 +74,12 @@
             </plugin>
         </plugins>
     </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-servlets</artifactId>
+            <version>9.4.28.v20200408</version>
+        </dependency>
+    </dependencies>
 </project>
```

Comment:
This might be more correct place to declare the dependency, but it is awfully lonely in pom.xml. Consider moving it to parent-pom.xml .